<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>474</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>479</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Emotion based music player</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #282828</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonPlayPause">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
 border-style: solid;
 border-width:1px;
 border-radius:37.5px;
 max-width:75px;
 max-height:75px;
 min-width:75px;
 min-height:75px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/play.png</normaloff>
          <normalon>icons/pause.png</normalon>icons/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSkip">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>77</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">
 border-style: solid;
 border-width:1px;
 border-radius:37.5px;
 max-width:35px;
 max-height:75px;
 min-width:75px;
 min-height:35px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/next.png</normaloff>icons/next.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSongName">
        <property name="minimumSize">
         <size>
          <width>371</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #FFFFFF</string>
        </property>
        <property name="text">
         <string>Nothing is playing...</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <item>
       <widget class="QLabel" name="labelInQueue">
        <property name="styleSheet">
         <string notr="true">color: #FFF</string>
        </property>
        <property name="text">
         <string>In queue: 0 songs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEmotion">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>No emotion... Wait...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid #1DB954;
    border-radius: 5px;
	height: 15px;
	color: #FFF;
}

QProgressBar::chunk {
    background-color: #1DB954;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="format">
       <string>0/0  </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="icons/images.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/images.qrc"/>
 </resources>
 <connections/>
</ui>
